cpr/cprtypes.cpp

```diff
@@ -4,20 +4,13 @@
 
 namespace cpr {
 
-bool CaseInsenstiveCompare::operator()(const std::string& a, const std::string& b) const {
-    return to_lower(a) < to_lower(b);
+bool CaseInsensitiveCompare::operator()(const std::string& a, const std::string& b) const noexcept {
+    return std::lexicographical_compare(a.begin(), a.end(),
+                                        b.begin(), b.end(), 
+                                        [](unsigned char ac, unsigned char bc)
+                                        {
+                                            return std::tolower(ac) < std::tolower(bc);
```

Comment:
Looks like you're missing a `#include <cctype>` for this function. See [this](http://en.cppreference.com/w/cpp/string/byte/tolower).